project-base/Makefile

```diff
@@ -12,8 +12,21 @@ generate-schema-native:
 	cd storefront; npm run gql
 	rm -rf storefront/schema.graphql
 
-run-acceptance-tests:
-	docker compose exec php-fpm php phing -D production.confirm.action=y -D change.environment=acc environment-change
+define run_acceptance_tests
+	docker compose exec php-fpm php phing -D production.confirm.action=y -D change.environment=test environment-change
```

Comment:
Why we do not run acceptance tests in acceptance environment?

Reply:
In the acceptance environment backend of the application overwrites its URL to webserver:8080, while the storefront has still the domain set to 127.0.0.1:8000. This causes improper functioning.